### ISSUE TITLE
16 Man Rosters

### ISSUE THREAD

**Issue by jackconnolly21** [michael-langaman/fntsylu#2]
How easy would it be to add functionality to 16-man rosters? (10 man lineup and 6 man bench)

**Comment by michael-langaman** [comment]
I don't believe it should be too difficult except that I can't test it since I don't have any leagues with 16-man rosters. I have a test league that I use for debugging but the deadline has passed to change league settings and such.

**Comment by jackconnolly21** [comment]
I changed all of the 13's to 16's and it seems to be working. Could possibly add in the option somewhere of what size team your league uses as a field to edit? In addition to username, password etc.

**Comment by michael-langaman** [comment]
That's a good option but I don't think it's necessary. I don't have code in front of me currently but I suppose changing the 16s to "self.getNumOfRows" should work. If not, adding a field for team size sounds good.
